BitwardenShared/UI/Platform/Settings/Settings/About/AboutProcessor.swift

```diff
@@ -81,9 +92,36 @@ final class AboutProcessor: StateProcessor<AboutState, AboutAction, Void> {
 
     /// Prepare the text to be copied.
     private func handleVersionTapped() {
-        // Copy the copyright text followed by the version info.
-        let text = state.copyrightText + "\n\n" + state.version
-        services.pasteboardService.copy(text)
+        var infoParts = [
+            state.copyrightText + "\n",
```

Comment:
🤔 Would it be better instead of doing string concatenation here, to just list the `"\n"` as another element in the array, since we're doing a bunch of concatenation later with the `.joined`? That would help make it more clear when reading what the expected output is vis-à-vis blank lines. Same would apply to all the newlines embedded in strings here.

Reply:
Tried it but the problem is that it adds an additional new line:
`copyright\n\n\n` instead of `copyright\n\n`
Because it'd be the `\n` between `copyright` and the `\n` element and then another `\n` between the `\n` element and `version`.

Reply:
🤦🏼‍♀️ Right, that makes sense. Could just do a `""` on a separate line, instead? But yeah, that's reasonable either way.